src/main/java/com/mifos/api/datamanager/DataManagerAuth.java

```diff
@@ -36,4 +36,10 @@ public DataManagerAuth(BaseApiManager baseApiManager) {
         return baseApiManager.getAuthApi().setFcmToken(userId, new FcmToken(token));
     }
 
+    public Observable<String> updatePassword(int userId, String password, String repeatPassword) {
+        UpdatePasswordPayload updatePasswordPayload = new UpdatePasswordPayload();
```

Comment:
make this class elements final and pass in constructor.

Reply:
done